src/tag_index.c

```diff
@@ -424,3 +424,44 @@ int TagIndex_RegisterType(RedisModuleCtx *ctx) {
 
   return REDISMODULE_OK;
 }
+
+// static size_t TagIndex_GetSuffixOverhead(TagIndex *idx) {
```

Comment:
Can be in separate PR?

Reply:
What do you mean? I left it as a comment so it won't be counted by coverage, and it can be used for debugging maybe.. I can remove it, but it may be useful for someone someday. I don't understand your meaning with a separate PR.

Reply:
Let's put it in a PR related to MOD-6865